utils/generate-bags/node-runtime/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "node-runtime-generate-bags"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license = "Apache-2.0"
+homepage = "https://substrate.io"
+repository.workspace = true
+description = "Bag threshold generation script for pallet-bag-list and zkv-runtime."
+publish = false
+
+[dependencies]
+zkv-runtime = { version = "0.5.1", path = "../../../runtime" }
+generate-bags = { version = "32.0.0", path = ".." }
```

Comment:
I'd suggest to skip the version tag (which is implicit) when you pass a local path